src/utils/i18n/i18n.ts

```diff
@@ -115,6 +143,12 @@ export class Messages {
 
         let val: string = this.resolveKey(this.curLang, key, nb, modifier);
 
+        if (this.globalParams
+            && (this.formatMode === FormatMode.Sprintf || this.formatMode === FormatMode.Vsprintf)
+        ) {
```

Comment:
Idealement la condition du if sur une seule ligne et on peut simplifier pour

`(this.formatMode === (FormatMode.Sprintf ||  FormatMode.Vsprintf))`

Reply:
Le changement a été apporté dans le dernier commit

Reply:
`(this.formatMode === (FormatMode.Sprintf || FormatMode.Vsprintf))` ne fonctionne pas, si la variable à la deuxième valeur (dans ce cas Vsprintf) ça retourne systématiquement faux.

https://jsfiddle.net/ap2fj5k0/4/